purbee_backend/backend_source/post/post_type.py

```diff
@@ -56,6 +56,7 @@ def has_created(self):
     def get_post_type_from_id(post_type_id):
         #this is a db method in database_utilities.py
         post_type_dictionary = get_post_type_from_post_type_id(post_type_id)
+        print(post_type_dictionary)
```

Comment:
I encourage you to remove print operations

Reply:
I remove this print statement and some other unnecessary print statements.